app/main.py

```diff
@@ -1,8 +1,47 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
```suggestion
    def __init__(
            self,
            comfort_class: int,
            clean_mark: int,
            brand: str
     ) -> None:
```

Reply:
Please follow this formatting throughout your code

Reply:
Made it in the next commit